### DISCUSSION TITLE
Filtering only for year on a Date type Custom Field

### DISCUSSION THREAD

**Discussion by victoriaronja** [netbox-community/netbox#13505] [Help Wanted!]
Hi everyone,

I've been wondering whether there is any way to filter for Devices that have a specific Custom Field of the Type "Date" to a specific year, rather than filtering for an exact date? For example, if I have a field XY, which is a Date, I would like to filter for all Devices that have the year of the Date set to 2024, while the month and day aren't important. Is there a way to do this? 
I'm currently still on v3.4.9 in case this is of any importance.

Thanks in advance.

**Comment by candlerb** [comment]
The following appears to work (3.5.6):

```
?cf_XY__gte=2024-01-01&cf_XY__lte=2024-12-31
```

You could use a custom link to generate this filter (e.g. `Devices expiring next year`)
